### PR TITLE
Calculate - changed

### PR DESCRIPTION
**Pull request by ploWoj**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @ploWoj

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ploWoj for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -3,5 +3,34 @@
 
 std::string calculate(const std::string& command, int first, int second) {
     // TODO: Implement your solution here and return proper value
-    return "";
+    if(command == "add"){
+        return std::to_string(first+second);
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:7]: Add space between operations like '+' '-' etc...

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -3,5 +3,34 @@
...
+    }
+    else if(command == "subtract"){
+        return std::to_string(first-second);
+    }
+    else if(command == "multiply"){
+        return std::to_string(first*second);
+    }
+    else if(command == "divide"){
+
+        if(second==0||first ==0){
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:17]: You can divide 0 / 10. It's 0 and this is correct answer :)

File: homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -3,5 +3,34 @@
...
+           
+
+
+   
+ 
+        
+
+         
+
+
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:34]: Remove redundant new lines

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**